ihe-interop/src/main/java/org/motechproject/ihe/interop/service/IHESendingTemplateService.java

```diff
@@ -0,0 +1,15 @@
+package org.motechproject.ihe.interop.service;
+
+/**
+ * Sending template service post xml data to HL7 recipient URL.
+ */
+public interface IHESendingTemplateService {
```

Comment:
IHETemplateService

Maybe in the future, there will be more methods in this service